openedx/adg/lms/webinars/admin.py

```diff
@@ -17,30 +12,9 @@ class WebinarAdmin(admin.ModelAdmin):
     """
     save_as = True
     exclude = ('is_cancelled',)
-    list_display = ('title', 'start_time', 'language', 'is_published', 'webinar_status')
+    list_display = ('title', 'start_time', 'language', 'is_published')
     search_fields = ('title',)
 
-    def webinar_status(self, webinar):
-        """
-        Method field to show webinar status.
-
-        Args:
-            webinar (Webinar): Current webinar object
-
-        Returns:
-            string: Webinar status either `Upcoming`, `Delivered`, `Cancelled` or `Draft`
-        """
-        if webinar.is_cancelled:
-            return WEBINAR_STATUS_CANCELLED
-
-        is_published = getattr(webinar, 'is_published', False)
-
-        if not is_published:
-            return WEBINAR_STATUS_DRAFT
-        elif webinar and webinar.is_upcoming_webinar:
-            return WEBINAR_STATUS_UPCOMING
-        return WEBINAR_STATUS_DELIVERED
-
```

Comment:
Any reason we are removing these?

Reply:
To simplify the admin side and restore it to default. We already have webinar separated based on status in the admin dashboard side.

Reply:
Actually this will make it easy to distinguish between webinars in django admin site so imo, let's not remove this.